packages/js/product-editor/src/components/manage-download-limits-modal/manage-download-limits-modal.tsx

```diff
@@ -6,22 +6,18 @@ import classNames from 'classnames';
 import { useInstanceId } from '@wordpress/compose';
 import { createElement, useState } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
-import {
-	BaseControl,
-	Button,
-	Modal,
-	// @ts-expect-error `__experimentalInputControl` does exist.
-	__experimentalInputControl as InputControl,
-} from '@wordpress/components';
+import { BaseControl, Button, Modal } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { ManageDownloadLimitsModalProps } from './types';
-import { useNumberInputProps } from '../../hooks/use-number-input-props';
+import { NumberControl } from '../number-control';
 
 const DOWNLOAD_LIMIT_MIN = 0;
+const DOWNLOAD_LIMIT_MAX = 10000000000;
 const DOWNLOAD_EXPIRY_MIN = 0;
+const DOWNLOAD_EXPIRY_MAX = 10000000000;
```

Comment:
You can use the underscore to make it more readable: 
```suggestion
const DOWNLOAD_EXPIRY_MAX = 10_000_000_000;
```